src/OpenTelemetry.AutoInstrumentation.Native/resource.h

```diff
@@ -1,8 +1,6 @@
 //{{NO_DEPENDENCIES}}
 // Microsoft Visual C++ generated include file.
```

Comment:
Did you have chance to regenerate it or modified manually?

Reply:
Good Q. I did it manually 😬 this file is generated by VS not a compilation step. Let me open `devenv.exe` and check if it is happy about it.

Reply:
The edition made via VS is slightly different than what I did manually but VS was happy with what is already in the PR. We can't require that everybody edit these files via VS but I can use the changes that it made instead of mine just so we keep it close to what VS is doing.